vllm/config/scheduler.py

```diff
@@ -37,6 +37,11 @@ class SchedulerConfig:
     This config has no static default. If left unspecified by the user, it will
     be set in `EngineArgs.create_engine_config` based on the usage context."""
 
+    prefill_max_num_batched_tokens: int | None = Field(default=None)
```

Comment:
If we are going to type hint `| None` then the `Field` is not necessary

```suggestion
    prefill_max_num_batched_tokens: int | None = None
```